task_lib.c

```diff
@@ -208,7 +212,10 @@ __attribute__ ((aligned(64))) void	*accept_cb_data;
 	void				(*connect_cb)(int64_t tfd, int result, void *connect_cb_data);
 	void				*close_cb_data;		//  User data to pass to the close callback
 	void				(*close_cb)(int64_t tfd, void *close_cb_data);
-	struct worker			*preferred_worker;	// To initiate task io worker migration
+	void				(*ssl_cb)(int64_t tfd, void *ssl_cb_data);
+	void				*ssl_cb_data;
+	SSL*				ssl;
+	struct worker		*preferred_worker;	// To initiate task io worker migration
```

Comment:
Please move `*ssl_cb` and `*ssl_cb_data` to below the `64 BYTE BOUNDARY` comment.  The `SSL *ssl` field can remain where it is.

I spent effort trying to ensure that the 64 byte cache line boundaries are clearly represented visually within the structure definitions, and the way it is now, there's 80 bytes in that block prior to the 64 BYTE BOUNDARY comment.

Overall, this change results in the cache-stride ordering become a 7/4 split, which is fine for staggered L2/L3 cache-line performance, so other than the visual change within the structure layout, this should be fine.